src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const HeaderStyle = {
+  color: '#b6ad90',
+};
+
+const Navbar = () => (
+  <nav>
+    <h1 style={HeaderStyle}>Bookstore</h1>
+    <ul className="nav_links">
+      <Link to="/"><li>Books</li></Link>
+      <Link to="/Catagories"><li>Categories</li></Link>
```

Comment:
- [ ] Change the link name to `Categories` not `Catagories` as declared in `App.js` otherwise you will get an error saying "route does not exist."